manager/integration/deploy/test.yaml

```diff
@@ -79,6 +79,8 @@ spec:
       value: "s3://backupbucket@us-east-1/backupstore$minio-secret, nfs://longhorn-test-nfs-svc.default:/opt/backupstore"
     - name: LONGHORN_BACKUPSTORE_POLL_INTERVAL
       value: "30"
+    - name: LONGHORN_DISK_TYPE
+      value: "ssd"
```

Comment:
How do think about this approach @yangchiu . If yes, can we modify the Jenkin pipeline to specify this env variable?

Reply:
LGTM.
And we've already had USE_HDD variable in the jenkins pipeline. We use it to specify this LONGHORN_DISK_TYPE value.

Reply:
Thanks